projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.html

```diff
@@ -49,14 +49,17 @@ <h1 class="h3 mb-3">{{ allDiagramOptions.title }}</h1>
 
 <ng-template #diagramTypeNav>
   <fhi-diagram-type-nav
-    *ngIf="allDiagramOptions.diagramTypeNavId !== undefined && allDiagramOptions.diagramTypeId !== undefined"
+    *ngIf="allDiagramOptions.diagramTypeId !== undefined"
     [currentDiagramTypeGroup]="currentDiagramTypeGroup"
     [currentDiagramTypeId]="allDiagramOptions.diagramTypeId"
     [currentMapTypeId]="allDiagramOptions.mapTypeId"
     [currentSeriesLength]="allDiagramOptions.series.length"
     (diagramTypeNavigation)="onDiagramTypeNavigation($event)"
   >
   </fhi-diagram-type-nav>
+
+  <!-- POC for new default nav -->
+  <!-- <fhi-diagram-type-nav-default></fhi-diagram-type-nav-default> -->
```

Comment:
@yleaxeman Uncomment line 62 to see all states in the new nav based on the new `DiagramTypeGroupService`